x/protorev/keeper/developer_fees_test.go

```diff
@@ -3,82 +3,160 @@ package keeper_test
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+
 	"github.com/osmosis-labs/osmosis/osmomath"
 	"github.com/osmosis-labs/osmosis/v23/app/apptesting"
 	"github.com/osmosis-labs/osmosis/v23/x/protorev/types"
 )
 
-// TestSendDeveloperFee tests the SendDeveloperFee function
-func (suite *KeeperTestSuite) TestSendDeveloperFee() {
+var (
+	usdcDenom = "usdc"
+	arbProfit = osmomath.NewInt(1000)
+)
+
+func (suite *KeeperTestSuite) TestDistributeProfit() {
 	cases := []struct {
 		description       string
 		alterState        func()
+		denom             string
 		expectedErr       bool
 		expectedDevProfit sdk.Coin
+		expectedBurnBal   sdk.Coin
+		expectedCommBal   sdk.Coin
 	}{
 		{
 			description:       "Send with unset developer account",
 			alterState:        func() {},
+			denom:             types.OsmosisDenomination,
 			expectedErr:       true,
 			expectedDevProfit: sdk.Coin{},
+			expectedBurnBal:   sdk.Coin{},
+			expectedCommBal:   sdk.Coin{},
 		},
 		{
 			description: "Send with set developer account in first phase",
 			alterState: func() {
 				account := apptesting.CreateRandomAccounts(1)[0]
 				suite.App.ProtoRevKeeper.SetDeveloperAccount(suite.Ctx, account)
 
-				err := suite.pseudoExecuteTrade(types.OsmosisDenomination, osmomath.NewInt(1000), 100)
+				err := suite.pseudoExecuteTrade(types.OsmosisDenomination, arbProfit, 100)
 				suite.Require().NoError(err)
 			},
+			denom:             types.OsmosisDenomination,
 			expectedErr:       false,
-			expectedDevProfit: sdk.NewCoin(types.OsmosisDenomination, osmomath.NewInt(20)),
+			expectedDevProfit: sdk.NewCoin(types.OsmosisDenomination, osmomath.NewInt(200)),
```

Comment:
ah numbers 10x'd because below we changed distribute profit call to be hard coded at 1000, just switching which denom,